ui/selectors/selectors.js

```diff
@@ -950,17 +951,17 @@ export function getDisabledRpcMethodPreferences(state) {
 }
 
 export function getShouldShowFiat(state) {
-  const isMainNet = getIsMainnet(state);
-  const isLineaMainNet = getIsLineaMainnet(state);
-  const isCustomNetwork = getIsCustomNetwork(state);
+  const currentChainId = getCurrentChainId(state);
```

Comment:
Perhaps a future enhancement is to support a chain ID argument so we're less coupled to the global network?

Reply:
That's right, better be handled standalone PR since it will require lot of prop drilling and test file changes.